README.md

```diff
@@ -89,6 +89,11 @@ The landing URI is where the middleware redirects the user when the
 authentication process is complete. This could just be back to the
 index page, or it could be to the user's account page.
 
+* `:creds-in-header?`
```

Comment:
`:basic-auth?` might be a better name for this option, since this option passes credentials through HTTP Basic Authentication.